test/functions_test.py

```diff
@@ -101,56 +101,80 @@
 Traceback (most recent call last):
         ...
 TypeError: edge_gen() missing 1 required positional argument: 'sign'
->>> input_dic=get_input(input_func=print_test)
->>> input_dic['sign']
-2
->>> input_dic['vertices']
-20
->>> input_dic['min_edge']
-20
->>> input_dic['min_weight']
-1
->>> input_dic['output_format']
-1
->>> input_dic['max_weight']
+>>> prev_item = ""
+>>> input_func_dict = {"vertices":"120","max_weight":"110","min_weight":"0","min_edge":"1","max_edge":"1000","sign":"1","direct":"1","self_loop":"1","multigraph":"1","file_name":"File 1","output_format":"2","weight":"1","error":"120"}
+>>> def input_func_test(input_data):
+...    menu = dict(MENU_ITEMS1,**MENU_ITEMS2)
+...    global prev_item
+...    for item in menu:
+...        if input_data == menu[item]:
+...            if item != prev_item :
+...                prev_item = item
+...                return input_func_dict[item]
+...            else:
+...                return input_func_dict["error"]
```

Comment:
why shouldn't we add this function to `functions.py` and then import it into this test file then use it when we need it?

Reply:
This function is only for **test** so I think it's not necessary to add it to `functions.py`